src/app/home/views/reviewUsers.ejs

```diff
@@ -6,7 +6,7 @@
 
         <script>
             $(function() {
-                loadReviewUsersBanner();
+                //loadReviewUsersBanner();
```

Comment:
remove script tag if do not require this function anymore

Reply:
this is not my work bud and was stopping me working so added comment do you want me to remove the commented out line

Reply:
added back after testing